README.uml

```diff
@@ -7,18 +7,20 @@
 # 1: when a class creates a list of other classes,
 # 2: when class names are passed by argument.
 
-# for the dynamo 0.1 api:
-pyreverse -A -p dynamo0p1 dynamo0p1.py # -A includes base classes
-dotty classes_dynamo0p1.dot # to view
-dot -Tsvg classes_dynamo0p1.dot > classes_dynamo0p1.svg # to generate svg
+# for the dynamo 0.3 api:
+pyreverse -A -p dynamo0p3 dynamo0p3.py # -A includes base classes
+dotty classes_dynamo0p3.dot # to view
+dot -Tsvg classes_dynamo0p3.dot > classes_dynamo0p3.svg # to generate svg
+dot -Tpng classes_dynamo0p3.dot > classes_dynamo0p3.png # to generate png
 
 # for gocean
 pyreverse -A -p gocean gocean.py
 dotty classes_gocean.dot
 dot -Tsvg classes_gocean.dot > classes_gocean.svg
 
-# svg can be imported into libreoffice, although there appears to be a bug with visualisation
-# (it looks OK in eog)
-
-# below might make it prettier?
-http://www.hokstad.com/making-graphviz-output-pretty-with-xsl
+# to make the topclasses.png file I manually edited the dot file to
```

Comment:
Thank you for explaining the editing process. Since it is a README documentation, think it would be better to write in 3rd person/passive voice. E.g. "to make the topclasses.png file manually edit the dot file to remove any unwanted nodes and edges then add ...."

Reply:
Reworded.